src/errors.js

```diff
@@ -34,19 +36,18 @@ class TooManyAccountsIOSError extends Error {
 
 class AmountTooSmallError extends Error {
     constructor() {
-       super('Amount is too small');
-       this.code = 'K5';
+        super('Amount is too small');
+        this.code = 'K5';
     }
 }
 
 class NetworkMissmatchError extends Error {
-
     /** @param {string} transactionNetwork
      *  @param {string} keyguardNetwork
      */
     constructor(transactionNetwork, keyguardNetwork) {
-       super(`Network missmatch: ${transactionNetwork} in transaction, but ${keyguardNetwork} in Keyguard`);
-       this.code = 'K6';
+        super(`Network missmatch: ${transactionNetwork} in transaction, but ${keyguardNetwork} in Keyguard`);
```

Comment:
What's the standard for template strings: `asfafs ${ var } asdfasd` or `asfafs ${var} asdfasd`. I prefer with spaces, it's more legible.